### ISSUE TITLE
Initial commit option

### ISSUE THREAD

**Issue by maximkoretskiy** [iamstarkov/generator-git-init#2]
It can be useful to add an ability to create an initial commit with your generator. 
What do you think about it? 
I can implement this if you are fine with it.

Btw nice and modular generator.

**Comment by iamstarkov** [comment]
hi, thank you for kind words.

**Comment by iamstarkov** [comment]
im also need this ability from time to time, so its one point for this idea

**Comment by iamstarkov** [comment]
two thing contra is that someone needs commit without init and it will be really confusing to use `yo-git-init` to be able to commit

**Comment by iamstarkov** [comment]
what do you think?

**Comment by maximkoretskiy** [comment]
Is the problem in generator name? Absolutely no. 
I can see some elegance in it:
```sh
> yo sp:git-init --commit
git init
git add .
git commit -m 'Init commit'
```
It seems that it`s too odd to have 2 generators for this purpose.

**Comment by maximkoretskiy** [comment]
@iamstarkov BTW PR is ready

**Comment by maximkoretskiy** [comment]
I had some issues with PR test passing. But now everything works fine.

**Comment by iamstarkov** [comment]
cool, will review today

**Comment by iamstarkov** [comment]
published as `v1.1.0`

**Comment by iamstarkov** [comment]
thank you
